src/boot/boot.js

```diff
@@ -45,11 +45,19 @@ function bootHypothesisClient(doc, config) {
   // Register the URL of the sidebar app which the Hypothesis client should load.
   // The <link> tag is also used by browser extensions etc. to detect the
   // presence of the Hypothesis client on the page.
-  var baseUrl = doc.createElement('link');
-  baseUrl.rel = 'sidebar';
-  baseUrl.href = config.sidebarAppUrl;
-  baseUrl.type = 'application/annotator+html';
-  doc.head.appendChild(baseUrl);
+  var sidebarUrl = doc.createElement('link');
+  sidebarUrl.rel = 'sidebar';
+  sidebarUrl.href = config.sidebarAppUrl;
+  sidebarUrl.type = 'application/annotator+html';
+  doc.head.appendChild(sidebarUrl);
+
+  // Registur the URL of the annotation client which is currently being used to drive
+  // annotation interactions. 
+  var clientUrl = doc.createElement('link');
+  clientUrl.rel = 'annotation-client';
+  clientUrl.href = config.assetRoot;
```

Comment:
In testing this I discovered that although this works in the dev server, it isn't going to work in production or the Chrome extension. `assetRoot` points to the root URL for the package's files. In development, the package server (in `serve-package.js`) happens to return the boot script when fetching this URL but that doesn't match the behaviour of cdn.hypothes.is or the Chrome extension.

You can get the URL of the boot script given `assetRoot` however: `config.assetRoot + 'build/boot.js` should return the URL of the boot script. This should I think then work automatically with the Chrome/Firefox extensions.